src/phases/evolution-phase.ts

```diff
@@ -33,12 +35,13 @@ export class EvolutionPhase extends Phase {
   protected pokemonEvoSprite: Phaser.GameObjects.Sprite;
   protected pokemonEvoTintSprite: Phaser.GameObjects.Sprite;
 
-  constructor(scene: BattleScene, pokemon: PlayerPokemon, evolution: SpeciesFormEvolution | null, lastLevel: integer) {
+  constructor(scene: BattleScene, pokemon: PlayerPokemon, evolution: SpeciesFormEvolution | FusionSpeciesFormEvolution | null, lastLevel: integer) {
```

Comment:
```suggestion
  constructor(scene: BattleScene, pokemon: PlayerPokemon, evolution: SpeciesFormEvolution | null, lastLevel: integer) {
```

Same thing, unless it is needed to prevent instanceof shenanigans or something.
I did my playtesting without those FusionSpeciesFormEvolution declarations and all seemed fine